tests/switch/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -54,6 +54,33 @@ switch (a) {
     if (1) {};
     c;
 }
+
+switch (a) {
+  case x:
+  case y:
+    call();
+
+  case z:
+    call();
+}
+
+switch (a) {
+  case x:
+
```

Comment:
Correct me if I'm wrong but isn't that part of the snapshot the "before format" and the other part "after format" and not using different parsers?